blog/2025-01-28-ratify-1.4.0.mdx

```diff
@@ -0,0 +1,56 @@
+---
+slug: ratify-v1.4.0-is-available
+title: Announcing Ratify v1.4.0 - Revocation Checking with CRL Support, Enhanced Out-of-box Experience, and New Cloud Provider Support
+authors: [ratify]
+tags: [verification]
+---
+
+We are thrilled to announce the release of [Ratify v1.4.0](https://github.com/ratify-project/ratify/releases/tag/v1.4.0)! This milestone release introduces significant new features that enhance Ratify's capabilities as a trusted supply chain security tool. As always, we deeply appreciate the contributions from the community, which make these advancements possible.
+
+## New Features
+
+### Revocation Checking with Certificate Revocation List (CRL) Support
+   
+Security in supply chain verification is critical, and this release marks an important step forward with the addition of revocation checking. Certificate revocation checking enhances security by ensuring that compromised or expired certificates are not used, thereby maintaining the integrity and trustworthiness of digital signatures. Ratify supports validating signing identity (certificate and certificate chain) revocation status. 

```

Comment:
Hi @FeynmanZhou , clarification on "compromised or expired certificates are not used". Can expired cert be validated with timestamping support?

Reply:
Yes. timestamping support in Ratify extends the trust of signatures created within the certificate's validity period by trusting on a Timestamping Authority (TSA), enabling successful signature verification even after the certificates have expired. You may want to refer to https://ratify.dev/blog/ratify-v1.3.0-is-available#support-of-validating-notary-project-signatures-with-timestamping